lib/internal/Magento/Framework/Validator/EmailAddress.php

```diff
@@ -9,4 +9,13 @@
 
 class EmailAddress extends \Zend_Validate_EmailAddress implements \Magento\Framework\Validator\ValidatorInterface
 {
+    public function isValid($value)
+    {
+        if (false === \strpos($value, '@')) {
```

Comment:
1) Please avoid Yoda conditions - JavaScript logic you wrote is much more readable ;) What would be really useful here is unit test
2) Remove leading backslash for function, there is no static test which forbids or enforces this but there is no other occurrence of `\strpos` currently. Let's be consistent.
3) `strrpos` will work faster than `strpos` at average as domain is usually shorter than account name ;D